application/cms/dimension_service.py

```diff
@@ -49,12 +49,12 @@ def update_dimension_chart_or_table(self, dimension, post_data):
         if "chartObject" in post_data:
             data["chart"] = post_data["chartObject"]
             data["chart_settings_and_source_data"] = post_data["source"]
-            data["chart_title"] = post_data["source"]["chartFormat"]["chart_title"]
+            data["chart_title"] = post_data["chartTitle"]
```

Comment:
Do these still need to be under the conditional `if "chartObject" in post_data` now that they aren't delving deep into the nesting?  I guess it still makes sense to only check for them if there is an associated object?

Reply:
Yeah, I think it makes sense to only check if there's an associated chart. Otherwise we'd either have to check that `chartTitle` is defined, or always end up with a `chart_title` (possibly `None`) in the resulting `data` dict.